network-store-model/src/main/java/com/powsybl/network/store/model/RegulatingPointAttributes.java

```diff
@@ -28,6 +28,9 @@ public class RegulatingPointAttributes extends AbstractAttributes implements Att
     @Schema(description = "Resource type of the regulating equipment")
     private ResourceType regulatingResourceType;
 
+    @Schema(description = "Resource sub type of the regulating equipment (for tap changer of transformer)")
+    private RegulatingTapChangerType regulatingResourceSubType;
```

Comment:
why is this sometimes called regulatingResourceSubType and also RegulatingTapChangerType?

Reply:
it will be RegulatingTapChangerType